api/src/main/java/org/pmiops/workbench/notebooks/LeonardoNotebooksClientImpl.java

```diff
@@ -70,6 +80,18 @@ private ClusterRequest createFirecloudClusterRequest(
     nbExtensions.put(
         "aou-activity-checker-extension", gcsPrefix + "/activity-checker-extension.js");
 
+    Workspace workspace = workspaceService.getRequired(workspaceNamespace, workspaceName);
+    Map<String, String> customClusterEnvironmentVariables = new HashMap<>();
+    // i.e. is NEW or MIGRATED
+    if (!workspace.getBillingMigrationStatusEnum().equals(BillingMigrationStatus.OLD)) {
```

Comment:
@calbach here is what I did for setting / not setting env vars based on migration status